content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -132,6 +132,8 @@ directories:
 ## Kubeflow installation
 Run the following commands to set up and deploy Kubeflow.
 
+> We strongly recommend to install kfctl v1.2 or above because kfctl v1.2 addresses several critical bugs that can break the Kubeflow deployment.
+
 1. Download the kfctl {{% kf-latest-version %}} release from the
   [Kubeflow releases 
   page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
```

Comment:
Currently, the kfctl installation step is below the recommendation to have v1.2+. Maybe it'd be best to say that v1.2 is 👍 after the user is learns about "downloading kfctl"? It seems logical. WDYT?

```suggestion
1. Download the kfctl {{% kf-latest-version %}} release from the
  [Kubeflow releases 
  page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
  
  **Note**: You're strongly recommended to install **kfctl v1.2** or above because kfctl v1.2 addresses several critical bugs that can break the Kubeflow deployment.
```

(with added emphasis in **bold** as a recommendation, since it looks like a critical step)